HW_1/task_4.py

```diff
@@ -0,0 +1,17 @@
+# Задача 4. Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов.
+# Сколько журавликов сделал каждый ребенок, 
+#если известно, что Петя и Сережа сделали одинаковое количество журавликов, а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+#*Пример:*
+# 6 -> 1  4  1
+#24 -> 4  16  4
+# 60 -> 10  40  10
+
+s=int(input("Введите количество журавликов, которое сделали все дети: "))
+if s >= 6 and s % 6 == 0:
```

Comment:
Без первого условия можно вполне обойтись
```suggestion
if s % 6 == 0:
```